pkgs/development/python-modules/pymediainfo/default.nix

```
@@ -48,6 +48,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/sbraz/pymediainfo";
     changelog = "https://github.com/sbraz/pymediainfo/releases/tag/v${version}";
     license = licenses.mit;
```

```suggestion
    license = lib.licenses.mit;
```

Nits: try to remove `with lib;`, minimize its scope is a good idea